lib/datadog/profiling/collectors/cpu_and_wall_time_worker.rb

```diff
@@ -40,7 +40,7 @@ def start
 
             @worker_thread = Thread.new do
               begin
-                Thread.current.name = self.class.name unless Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.3')
```

Comment:
Wondering if we should have a helper or two to do these kind of checks instead of ad-hoc winging the checks each time.

This would also make such calls log-able/trackable/grep-able.

Reply:
I would usually say yes, but given that's additional work done just for the benefit of Ruby 2.1 and 2.2, in this very specific case I don't think it's work the extra effort.